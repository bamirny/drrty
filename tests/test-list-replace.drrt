def int main(){
	list[int] numbers;
	numbers = [1,2,3,4,5,6,7];
	print(numbers.get(3));

	numbers.set(3,44);    /*test replaces values in the list*/
	print(numbers.get(3));

	return 0;
		



}