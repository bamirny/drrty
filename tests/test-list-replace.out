4<br>
44<br>
